File: src/test/java/ar/edu/unlam/tallerweb1/infrastructure/DataBaseConnectionTest.java
package ar.edu.unlam.tallerweb1.infrastructure;

import org.junit.jupiter.api.Test;

import java.sql.SQLException;

import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.fail;

public class DataBaseConnectionTest extends JdbcRepositoryTest{

    @Test
    public void checkDbConnection(){
        try {
            assertFalse(connection.isClosed());
        } catch (SQLException e) {
            fail(e);
        }
    }
}

File: src/test/java/ar/edu/unlam/tallerweb1/infrastructure/DataBaseConnectionTest.kt
package ar.edu.unlam.tallerweb1.infrastructure

import org.junit.jupiter.api.Assertions.assertFalse
import org.junit.jupiter.api.Assertions.fail
import org.junit.jupiter.api.Test
import java.sql.SQLException

class DataBaseConnectionTest : JdbcRepositoryTest() {
    @Test
    fun checkDbConnection() {
        try {
            assertFalse(connection.isClosed)
        } catch (e: SQLException) {
            fail<Any>(e)
        }
    }
}
